docs/try_local_system.md

```diff
@@ -338,6 +371,18 @@ Once you're done with configuring the application, proceed with
    2. Add the merchant ID you obtained in the previous step under the "current
       value" column for the `merchant_id` variable.
 
+#### Local Postman Variables 
+
+|variable|default|current|
+|:------:|:-----:|:-----:|
+|baseUrl||http://localhost:8080|
+|admin_api_key||test_admin|
+
+NOTE :- 
+To test stripe add stripe secret key to variable `connector_api_key`.
+eg. `sk_test_5142342342323iuh2b34jb2jh3432`  
```

Comment:
Hi, Is this a sample key or a actual api-key of `stripe`. If this is a actual key remove this.